SignupPage.java

```diff
@@ -0,0 +1,13 @@
+public class SignupPage {
+	private String username;
+	private String password;
+
+	public SignupPage(String username, String password){
+		this.username = username;
+		this.password = password;
+	}
+
+	public void addAccount() {
+		# TODO
```

Comment:
You left this as a TODO, please complete so I can approve.

Reply:
Done!